client/src/featureform/register.py

```diff
@@ -289,6 +289,40 @@ def register_redis(self,
         self.__resources.append(provider)
         return OnlineProvider(self, provider)
 
+    def register_firestore(self,
+                       name: str,
+                       description: str = "",
+                       team: str = "",
+                       collection: str = "",
+                       project_id: str = "",
+                       credentials: bytearray = []
```

Comment:
Change this to accept a file location, then json.dumps the contents of the file as the parameter for the FirestoreConfig